generate_release_cycle.py

```diff
@@ -0,0 +1,110 @@
+"""
+Read in a JSON and generate two CSVs and a Mermaid file
+"""
```

Comment:
```suggestion
"""Read in a JSON and generate two CSVs and a Mermaid file."""
```

Two CSVs, a JSON and a Mermaid walk into a bar...

(Trivial nit, but PEP 257, the docstring standard, requires docstring summary lines to end with a period (as they are complete sentences), and says that for one line docstrings, the starting and ending triple-quotes should be on the same line.)